otter/convergence/service.py

```diff
@@ -607,7 +607,7 @@ def buckets_acquired(self, my_buckets):
         """
         ceff = Effect(GetChildren(CONVERGENCE_DIRTY_DIR)).on(
             partial(self._converge_all, my_buckets))
-        return perform(self._dispatcher, self._with_conv_runid(ceff))
+        return (perform(self._dispatcher, self._with_conv_runid(ceff)), )
```

Comment:
Ok cool, thanks.  :)  Would it make sense to include that as a comment, or in the docstring (e.g. the return value only matters in that if a deferred is returned, the partitioner service will not stop until the deferred is callbacked?